src/test/java/hudson/plugins/git/GitSCMTest.java

```diff
@@ -3064,6 +3064,24 @@ public void testCommitMessageIsPrintedToLogs() throws Exception {
         assertThat(values, hasItem("Commit message: \"test commit\""));
     }
 
+    @Test
+    public void testCommitMessageIsEnvVar() throws Exception {
```

Comment:
Similar to this I was trying to set environment variable for https://issues.jenkins.io/browse/JENKINS-38699. Does this test case run correctly in the current state?